src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp

```diff
@@ -15,16 +15,210 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-/* ScriptData
-SDName: Eversong_Woods
-SD%Complete: 95
-SDComment: Quest support:
-SDCategory: Eversong Woods
-EndScriptData */
+#include "Common.h"
+#include "ScriptMgr.h"
+#include "ScriptedCreature.h"
 
-/* ContentData
-EndContentData */
+enum Partygoer_Pather
+{
+    EVENT_PATH                    = 1,
+    EVENT_RANDOM_ACTION_PATHER    = 2,
+    EVENT_REMOVE_EQUIPMENT_PATHER = 3,
+    EVENT_STOP_DANCING_PATHER     = 4
+};
+
+struct npc_partygoer_pather : public ScriptedAI
+{
+    npc_partygoer_pather(Creature* creature) : ScriptedAI(creature)
+    {
+        Initialize();
+    }
+
+    void Initialize()
+    {
+        _path = 594440;
+    }
+
+    void Reset() override
+    {
+        _events.ScheduleEvent(EVENT_RANDOM_ACTION_PATHER, urand(11000, 14000));
+    }
+
+    void PathEndReached(uint32 pathId) override
```

Comment:
/* pathId */ maybe will have it pass the ci. Did something simular on the anticheat before.